liquibase/scripts/fix_liquibase_snapshots.sh

```diff
@@ -17,6 +17,9 @@
 #  The updated snapshots are written to the openmrs-core/liquibase/snapshots folder.
 #
 
-openmrs_version=2.4.0-SNAPSHOT
+pom_path="/mnt/c/Users/Admin/Desktop/development/openmrs-dev/openmrs-core/liquibase/pom.xml" 
```

Comment:
I'd like to suggest using relative paths instead absolute path. Absolute paths can lead to issues when the project is moved to a different directory or when others are working on the project from different environments.